pyproject.toml

```diff
@@ -55,6 +55,7 @@ asyncpg = { version = ">=0.28,<0.30", optional = true }
 psycopg2-binary = { version = "^2.9.1", optional = true }
 mysqlclient = { version = "^2.1.0", optional = true }
 PyMySQL = { version = "^1.1.0", optional = true }
+setuptools = "^75.1.0"
```

Comment:
Why do we need this?

Reply:
CodeFactor failed the prior commit due to https://github.com/advisories/GHSA-cx63-2mw6-8hw5